pygmt/figure.py

```diff
@@ -393,7 +393,7 @@ def show(self, dpi=300, width=500, method=None, waiting=0.5, **kwargs):
                 )
             )
 
-        if method in ["notebook", "none"]:
+        if method == "notebook":
```

Comment:
Note to self that `None` != `"none"`. Probably should call `"none"` `"disable"`, but let's not break backwards compatibility.